src/inputs/labeled-inputs.scss

```diff
@@ -169,6 +167,45 @@
   }
 }
 
+/// Wrapping
+@mixin iui-input-label-wrapper {
+  @include iui-input-label-styling;
+  grid-area: label;
+  align-self: center;
+}
+
+/// Non-wrapping
+@mixin iui-input-label {
+  @include iui-input-label-styling;
+  display: block;
+  width: fit-content;
```

Comment:
Do you really need these?

Reply:
`display: block` and further down `display: inline-block` can be safely removed now that we have the grid.

`width: fit-content` however would be nice to keep. Without it you get the pointer cursor when hovering anywhere in this highlighted area.
![Screen Shot 2021-10-18 at 5 39 36 PM](https://user-images.githubusercontent.com/849817/137810538-a0ac80c6-0d7a-49cb-b823-b15ab7392781.png)

Reply:
> Without it you get the pointer cursor when hovering anywhere in this highlighted area.

It seems this is only happening because of `display: grid` (which we don't need for block layout) so it could be removed

Reply:
> display: block and further down display: inline-block can be safely removed now that we have the grid.

I think people should be allowed to use this component without grid, so maybe we should add back `display: inline-block` (without it, label defaults to `display: inline`).

Reply:
Removed `width: fit-content` and replaced with `display: inline-block`.